suite-native/message-system/src/components/FeatureMessageScreen.tsx

```diff
@@ -126,7 +126,7 @@ export const FeatureMessageScreen = () => {
                     </Button>
                 )}
                 {isDismissable && (
-                    <Button size="large" onPress={handleDismiss}>
+                    <Button size="large" colorScheme="tertiaryElevation0" onPress={handleDismiss}>
```

Comment:
According to design, this is wrong. Why did you change this?

![image](https://github.com/trezor/trezor-suite/assets/36101761/55025ae7-ab95-419c-a319-26be9b3a2235)